### PR TITLE
Without outer bracket [] the kit will not be detected

### PR DESCRIPTION
**Pull request by Soily**
abe8757ddb5a3904023d16d13844c00178fc2415

The following changes are proposed:

Add outer main brackets to code-snippets.

## Other Notes/Information
Tested on windows. Without the outer main brackets, the cmake-kit will not be detected.

### CODE REVIEW

File: docs/kits.md [hunk 1]
```diff
@@ -74,25 +74,29 @@ To specify a compiler, list the path to the compiler in the `cmake-tools-kits.js
 The most common CMake languages are `C` and `CXX`, and CMake Tools has built-in support for finding these. However, any language can be specified:
 
 ```xml
```
Comment (bobbrow) [inline docs/kits.md:76]: Would you mind changing the language specifier to `json` while you're in this file?

Comment (Soily) [reply docs/kits.md:76]: Hi there, 
I already have json selected (it was automatically recognized!). Or what else could I do? 

It's showing me this:
image


So when I remove the [] the cmake-kit will not be detected. It is always reproducable.

Comment (bobbrow) [reply docs/kits.md:76]: What I meant was that language selected for this code block is `xml` and it should be changed to `json`
image

Comment (Soily) [reply docs/kits.md:76]: Hi Bob,

I still don't get it sorry. I'm in the cmake-tools-kits.json-file and I have chosen below for language mode "JSON" of course, why should I have it set to xml? It doesn't make any sense to me.

???

Kind Regards
Paddy

Comment (bobbrow) [reply docs/kits.md:76]: Don't worry about it. I'll fix it.

I'm talking about this file in the PR.  `docs/kits.md` 
You updated the code blocks that are inside the triple backticks.  There is a language selector on the triple backticks that was incorrectly set to xml years ago (not in your PR).  I was just going to have you fix it while you have the file open for editing, but I can update your branch and push the change.

**Review by bobbrow — APPROVED** [review]

**Comment by Soily** [comment]
@microsoft-github-policy-service agree

**Pull request merged**